examples/with-shopify/components/cart-modal.js

```diff
@@ -0,0 +1,90 @@
+import Modal from 'react-modal'
+import { useCart, useCheckout } from '@/lib/cart'
+import CartItem from './cart-item'
+import Button from './button'
+import styles from './cart-modal.module.css'
+
+Modal.setAppElement('#__next')
```

Comment:
Minor: Can we wrap this side effect within a `useEffect, []` or when the app bootstraps?

Reply:
Tested and yes!

Reply:
Done 👍